lib/rubygems/requirement.rb

```diff
@@ -51,7 +51,11 @@ class BadRequirementError < ArgumentError; end
   # If the input is "weird", the default version requirement is
   # returned.
 
-  def self.create input
+  def self.create *inputs
+    return create inputs if inputs.length > 1
```

Comment:
Why not `new inputs`? This should match [L63](https://github.com/rubygems/rubygems/pull/1830/files#diff-d35975a5cb9b7bd38d28301c7ef405b1R63) right?